.gitignore

```diff
@@ -24,3 +27,7 @@ README.md
 *pytest_cache*
 *.log
 compile_options.sh
+./trunk/NDHMS/Land_models/NoahMP/MPP
```

Comment:
This file needs cleaned up. The only one that I can obviously approve of is
trunk/NHDMS/setEnvar.sh.
If you want to some of these to the respository for everyone, please give some justification.

Reply:
You are right. My mistake. I will clean this up.

Reply:
Except for setEnvar.sh (which I love adding). The things added are all symlinks.